### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by Akanksha987**
->ER Diagram and Schema Diagram
->Loom Video for the explanation of the PR.
https://www.loom.com/share/46c615b9428e4d598e4037feedfe84e1?sid=d8da5e54-0c91-4da3-9ad0-2d5861c57217
https://www.loom.com/share/5d1938d7c4904d808aea8ede9dcda758?sid=8f8e977c-7512-4b23-8f28-f0c420afc62e

### CODE REVIEW

**Comment by th3c0d3br34ker** [comment]
hey @Akanksha987, thanks for your submission. you have explained your ERD and Schema. 

However, the `GuestStays` entity doesn't fit in this scenario. Please schedule a catchup with me.

**Comment by Akanksha987** [comment]
Sure

**Comment by Akanksha987** [comment]
Hey can you please share your calendar so that I can schedule the catchup

**Comment by th3c0d3br34ker** [comment]
please schedule [here](https://calendar.kalvium.com/jainam-desai/30min).

**Comment by th3c0d3br34ker** [comment]
hi @Akanksha987, it seems there were some during the video call from your side.

as i understand, you joined the call and shared your screen, but choose not to respond to my queries. this was continued for 10 mins.

i have no choice but to escalate this issue.

thanks.

**Comment by Akanksha987** [comment]
Hey @th3c0d3br34ker  I have updated the ERDiagram and Schema Diagram as per your suggestion.

File: Schemas.png
Comment (th3c0d3br34ker) [inline Schemas.png:1]: hey @Akanksha987 the connection between `Reservation` and `Room` here seems inconsistent.

the `GuestID` property in `Reservations` is connecting to the `Rooms` entity. please consider to recheck the connections made.

**Review by th3c0d3br34ker — CHANGES_REQUESTED** [review]
thanks @Akanksha987, the ERD looks good to go. please consider rechecking the schema.

**Comment by Akanksha987** [comment]
I think you misunderstood the connection as I created er diagram there were RoomId and GuestId in the reservation entity and RoomId is connected to Reservation and Guests is also connected to reservation but there is no connection between the room entity and guest entity. Similarly, I had done with a Schema Diagram and this schema diagram is created using MySQL code so I think there is no wrong connection between them.

**Comment by th3c0d3br34ker** [comment]
thanks @Akanksha987 for reverting back. the ERD is modified as per the discussion during viva.

however the schema is not consistent with what was discussed.

image

i was referring to the following connections made, in my previous comment.
`GuestID` should link to the `Guest` entity
`RoomID` should link to `Room` entity.


please consider the above points and let me know.

**Comment by Akanksha987** [comment]
I think because of the entity table you are confused I will just change the position of the table and do the commit

**Comment by Akanksha987** [comment]
Hey @th3c0d3br34ker, I have updated the Schema Diagram can you please check it once.

**Review by th3c0d3br34ker — APPROVED** [review]
thanks @Akanksha987 for the update, its clear now. 

please go ahead and merge the PR.

**Pull request merged**